presidio-anonymizer/presidio_anonymizer/anonymizer_engine.py

```diff
@@ -95,7 +92,26 @@ def anonymize(
 
         operators = self.__check_or_add_default_operator(operators)
 
-        return self._operate(text, merged_results, operators, OperatorType.Anonymize)
+        return self._operate(text=text,
+                             pii_entities=merged_results,
+                             operators_metadata=operators,
+                             operator_type=OperatorType.Anonymize)
+
+    def add_anonymizer(self, anonymizer_cls: Type[Operator]) -> None:
+        """
+        Add a new anonymizer to the engine.
+
+        anonymizer_cls: The anonymizer class to add to the engine.
+        """
```

Comment:
As this is kind of "special" operation, do we want to add a log here?

Reply:
Good point

Reply:
Fixed